Makefile

```diff
@@ -7,7 +7,7 @@
 #                                                                           |
 #                                                    +-------+              v
 #                                                    | Felix |   +---------------------+
-#                                                    |  Go   |   | calico-build/golang |
+#                                                    |  Go   |   | calico/go-build     |
```

Comment:
Agree, that would be better but this container is already in use so it's not easy to change.